src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
-body {
-  margin: 0;
+@import "./src/styles/reset.scss";
+@import "./src/styles/_variables";
+@import "./src/styles/blocks/stars.scss";
+@import "./src/styles/blocks/card.scss";
+@import "./src/styles/blocks/header.scss";
+@import "./src/styles/_extends";
+
+.main__container {
+  width: $main-width;
+  display: flex;
+  padding: 50px 40px;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
 }
```

Comment:
Move this code to a new `main.scss` file in blocks folder.
Use nesting.
`.main {
    &__container {`